ChaosMod/Effects/db/Misc/MiscUTurn.cpp

```diff
@@ -36,10 +38,31 @@ static void OnStart()
 	SET_GAMEPLAY_CAM_RELATIVE_HEADING(camHeading);
 }
 
+// effect by Reguas
+
+static void OnStartFake()
+{
+	OnStart();
+
+	GetComponent<EffectDispatcher>()->OverrideEffectNameId("misc_fakeuturn", "misc_uturn");
```

Comment:
Should check for the existence of the component with ComponentExists first
```suggestion
	if (ComponentExists<EffectDispatcher>())
	{
		GetComponent<EffectDispatcher>()->OverrideEffectNameId("misc_fakeuturn", "misc_uturn");
	}
```